FreeRTOS/FreeRTOS-Kernel
# SMP updates

Description
-----------
1. Corrected memory allocation in Static Allocation for minimal task.
2. Uncrustified the output
3. Corrected bitmask errors in affinity
4. Using Core affinity instead of Core exclusion.

Test Steps
-----------
Tested with FreeRTOSConfig options:
1. configStaticAllocation (0,1)
2. configNUM_CORES (1,2,3,7)
3. configUSE_CORE_AFFINITY (0,1)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
